examples/vision/adamatch.py

```diff
@@ -69,7 +69,7 @@
 from tensorflow import keras
 from tensorflow.keras import layers
 from tensorflow.keras import regularizers
-from official.vision.image_classification.augment import RandAugment
+from official.vision.ops.augment import RandAugment
```

Comment:
Prefer KerasCV's RandAugment.

```
augmenter = RandAugment(num_layers=2, magnitude=5)
```

can be replaced with:

```
augmenter = keras_cv.layers.RandAugment(layers=3, magnitude=0.5)
```

usage:
```
images = augmenter(images)
```

Reply:
Thank you Luke, I have adapted the example to KerasCV